src/cryptography/hazmat/bindings/openssl/ssl.py

```diff
@@ -591,6 +595,18 @@
 static const long Cryptography_HAS_COMPRESSION = 1;
 #endif
 
+#if defined(SSL_CTRL_GET_SERVER_TMP_KEY)
+static const long Cryptography_HAS_GET_SERVER_TMP_KEY = 1;
+#undef SSL_get_server_tmp_key
+long SSL_get_server_tmp_key(SSL *ssl, EVP_PKEY **key) {
+    return SSL_ctrl(ssl, SSL_CTRL_GET_SERVER_TMP_KEY, 0, key);
+}
```

Comment:
Why are you redefining the macro? You should just be able to bind it, same as the other macros.